python/cuml/test/dask/test_random_forest.py

```diff
@@ -302,8 +302,8 @@ def test_rf_classification_dask_fil_predict_proba(partitions_per_worker,
         sk_mse = mean_squared_error(y_proba, sk_preds_proba)
 
         # The threshold is required as the test would intermitently
-        # fail with a max difference of 0.022 between the two mse values
-        assert fil_mse <= sk_mse + 0.022
+        # fail with a max difference of 0.0335 between the two mse values
+        assert fil_mse <= sk_mse + 0.0335
```

Comment:
This is concerning... surprised to see an increase in accuracy gap here? Any ideas on where it could come from? I believe for FIL we still test to a very tight threshold, right, so it should not be coming from inference?

Reply:
We no longer need to modify this test.